README.md

```diff
@@ -21,3 +21,10 @@
 1. Which programming, scripting, and markup languages are used to develop programs in the last years?
 2. What are the most utilized database environments for development projects in the past year?
 3. Is there a discernible correlation between the size of companies and the adoption of specific AI-powered search tools for business operations?
+
+### Zia
+1. Are professional developers using artificial intelligence tools in their development process ?
+2. What is the biggest benefit that developers perceive from AI tools, and what are the secondary benefits that are closely associated with them?
+3. What is the biggest benefit that developers perceive from AI tools, and what are the secondary benefits that are closely associated with them?
```

Comment:
your 2nd and 3. questions are just duplicated. please delete one of them.

Reply:
Thanks a lot for your comment ! I will fix it.

Reply:
Well Done Zia.